### PR TITLE
Removed Outdated, Skipped v0.12 Tests

### PR DESCRIPTION
**Pull request by rauljordan**
No tracking issue, this PR removes tests that are no longer relevant for v0.12.1 and are covered by spec tests

### CODE REVIEW

File: beacon-chain/core/state/benchmarks_test.go [hunk 1]
```diff
@@ -16,28 +16,6 @@ import (
 
 var runAmount = 25
 
-func TestBenchmarkExecuteStateTransition(t *testing.T) {
```
Comment (terencechain) [inline beacon-chain/core/state/benchmarks_test.go:19]: We shouldn't delete this.

CC @0xKiwi to regen the files

Comment (rauljordan) [reply beacon-chain/core/state/benchmarks_test.go:19]: @terencechain that benchmark is basically duplicated below. Theres the same benchmark with a full block right below it which passes correctly.

Comment (terencechain) [reply beacon-chain/core/state/benchmarks_test.go:19]: This is a test though. Below `BenchmarkExecuteStateTransition_FullBlock` is a benchmark which doesn't get run during test

Comment (rauljordan) [reply beacon-chain/core/state/benchmarks_test.go:19]: Fixing it up, nice catch- the naming is weird

**Pull request merged**